pkg/config/loader.go

```diff
@@ -72,7 +72,7 @@ func (e *EnvConfig) Load() error {
 		}
 		logging.S().Infof(".env.toml loaded from: %s", f)
 	} else {
-		logging.S().Warnf("no .env.toml found at %s; system may be misconfigured", f)
```

Comment:
I think this is a warning. If there is no `.env.toml` we are not inferring the `region` for AWS for example, so using `cluster:k8s` would fail.

Reply:
It really depends on the use case. For example, I am not running cluster:k8s, just small-scale tests with local:docker, and defaults are fine for me. Maybe we can display the warning on the cluster:k8s runner only. 

What I really wouldn't like to do is to greet every single new user of testground with a warning.

Reply:
> What I really wouldn't like to do is to greet every single new user of testground with a warning.

You have a point there.